### ISSUE TITLE
Add Codewars to list of social media

### ISSUE THREAD

**Issue by CodingSpecies** [EddieHubCommunity/LinkFree#229]
Is it possible to add codewars to the list of social media? Thank you.

**Comment by ShlokMohanty** [comment]
@CodingSpecies  are y9ou talking about the below one:

![image](https://user-images.githubusercontent.com/59838234/133818930-515a334a-263b-4e7a-861b-b3355280fa78.png)

**Comment by CodingSpecies** [comment]
yes @ShlokMohanty you can follow people here, and you do coding challenges

**Comment by ShlokMohanty** [comment]
@CodingSpecies yeah thats a good idea but like if you could give me the reason how it would help in adding to this linkfree like what  the purpose of adding the codewars here in the linkfree?

**Comment by CodingSpecies** [comment]
@ShlokMohanty CodeWars is where people create questions, and code tests, where you go to solve, and get ranked higher. It is a bit similar to GitHub as people create, star, fork and follow each other and the work they do. People can also sync their github profiles to CodeWars.

On linkfree, it could have a codewars option, where it takes you to the profile, where you can follow others.

Like the github option in linkfree, I thought this might be another idea.

It isn't so important, but just an idea, that I thought I could share. Thank you.

**Comment by eddiejaoude** [comment]
Yes we should add this to the social links option 👍

**Comment by san-coding** [comment]
I would love to work on this issue, i am new to open source ?

**Comment by Panquesito7** [comment]
> I would love to work on this issue, i am new to open source

Feel free to work on this issue. Let us know if you need any help here or in our [**Discord server**](http://discord.eddiehub.org). 🙂

**Comment by CodingSpecies** [comment]
@san-coding feel free to work on it!

**Comment by san-coding** [comment]
do you mean adding an option for code wars here
<img width="577" alt="image" src="https://user-images.githubusercontent.com/65719940/134972786-65c04824-2b2c-4129-8335-ae8734750f21.png">

**Comment by san-coding** [comment]
@CodingSpecies does `primeicons` have code wars icon? and what hex color should I use ?

**Comment by CodingSpecies** [comment]
@san-coding I am not sure for the color, but this is the icon: 

![image](https://user-images.githubusercontent.com/70807500/135109170-6d7559f1-8624-436c-8364-259b19e4e5c7.png)

**Comment by CodingSpecies** [comment]
@san-coding i don't think primeicons do as i tried to search it there, and it didn't come up. Thanks.

Also, I am not a maintainer, so I can't assign you, maybe @Panquesito7 can? Thank you.

**Comment by san-coding** [comment]
Thanks, since it is not there in can we add the link? will it display any logo for code wars ? or would adding the hex be enough for now

**Comment by Panquesito7** [comment]
> Thanks, since it is not there in can we add the link? will it display any logo for code wars ? or would adding the hex be enough for now

That depends. If CodeWars is provided in that library, then yes, it is just by adding the hex color. 🙂

**Comment by san-coding** [comment]
Yeah, I just checked, unfortunately `codewars` logo is not present in [primefaces/primeicons](https://github.com/primefaces/primeicons/tree/master/raw-svg)

**Comment by ankityadavhere** [comment]
Hey, [React Icons](https://react-icons.github.io/react-icons) is a popular icon library. Can this is considered?
As it has almost all social media icons!

If you haven't done the pr yet, lemme know if I can do this for hacktoberfest 👀

**Comment by san-coding** [comment]
> Hey, [React Icons](https://react-icons.github.io/react-icons) is a popular icon library. Can this is considered? As it has almost all social media icons! [codewars icon](https://react-icons.github.io/react-icons/search?q=codewar)
> 
> If you haven't done the pr yet, lemme know if I can do this for hacktoberfest 👀

Hey, @ankityadavhere , I am waiting for `hacktoberfest` to begin to open the PR for this👀

**Comment by ankityadavhere** [comment]
So I can't make this PR :'(

Wish you luck 🚀

**Comment by CodingSpecies** [comment]
I am not sure if this repo is signed up for hacktoberfest.

**Comment by san-coding** [comment]
@ankityadavhere there are plenty of issues out there, you can contribute to my open source project [ShopOnline](https://github.com/sudhay23/ShopOnline-Web-App/issues), we are participating in hacktoberfest

**Comment by deephunt3r** [comment]
> I am not sure if this repo is signed up for hacktoberfest.

Yeah! Eddie added `hacktoberfest` label to this issue 🎉

**Comment by CodingSpecies** [comment]
> > I am not sure if this repo is signed up for hacktoberfest.
> 
> Yeah! Eddie added `hacktoberfest` label to this issue 🎉

Thank you for clarifying @deephunt3r !

**Comment by san-coding** [comment]
I will add the the PR with the appropriate hex code

**Comment by ankityadavhere** [comment]
> @ankityadavhere there are plenty of issues out there, you can contribute to my open source project [ShopOnline](https://github.com/sudhay23/ShopOnline-Web-App/issues), we are participating in hacktoberfest, sorry for plugging it in here.

I'm coming ...🚀 Awesome project! @san-coding

**Comment by san-coding** [comment]
> > @ankityadavhere there are plenty of issues out there, you can contribute to my open source project [ShopOnline](https://github.com/sudhay23/ShopOnline-Web-App/issues), we are participating in hacktoberfest, sorry for plugging it in here.
> 
> I'm coming ...🚀 Awesome project! @san-coding

Thanks @ankityadavhere

**Comment by amittri1025** [comment]
#394 Changed Icon Library to **FontAwesome**

**Comment by krishguptadev** [comment]
We are still choosing a icon library, currently that icon is not supported so this issue can be closed

**Comment by CodingSpecies** [comment]
Is there any updates on this? Thank you.
